pages/guides/auth/client-credentials-flow.mdx

```diff
@@ -78,9 +78,11 @@ application_token = $dwolla.auths.client
 
 A refresh token is not paired with an application access token, therefore in order to refresh authorization you'll simply request a new application access token by exchanging your client credentials (as shown above).
 
-That's it! You're ready to start [making requests](/guides/auth/using-an-access-token.html) to the [Dwolla API](https://docs.dwolla.com/) on behalf your application.
+That's it! You're ready to start [making requests](/guides/auth/using-an-access-token) to the [Dwolla API](https://docs.dwolla.com/) on behalf your application.
 
-<nav class="pager-nav">
-  <a href="./">Back: Overview</a>
-  <a href="using-an-access-token.html">Next step: Using an access token</a>
-</nav>
+<Paginaton variant="previous" variantText="PREVIOUS" href="./"></Paginaton>
```

Comment:
I believe Connor added an implementation for <Pagination> that dynamically adds the NEXT and BACK pagination for each doc in a group. https://github.com/Dwolla/dev-portal/blob/master/layouts/guides.tsx#L39
Ahhh I hate to say this after you're put in so much work into adding these to each page already! :(
We may need to add a create a cleanup ticket to remove these pagination components from the docs.

Reply:
Yep, that is the case. We don't need to add the Pagination components at all within the MDX files. Sorry that I wasn't clear about that! 🙁

Reply:
Didn't spend too much time, just a lot of copy pasting ha! I can update in the next PR

Reply:
Sweet! Thanks :) Everything else looks great! 🙌